### DISCUSSION TITLE
Inline fractions

### DISCUSSION THREAD

**Discussion by trya2l** [typst/typst#3997] [Q&A] [answered]
Hi,

Following #3801, I would like to know if there is a "good way" to display fraction inline, literally like `a/b`.
Currently I am using this trick `$a\/b$` to display them inline.

**Comment by Enivex** [comment]
That is the way, though it's not unlikely that `/` will suffice inline in the future.

**Reply by trya2l** [reply]
That would be interesting. Thanks!

**Comment by MDLC01** [comment, marked as answer]
Note that you can also use `slash` : ` $a slash b$` .

**Reply by trya2l** [reply]
Oh, cool! Thanks
